ocs_ci/templates/ocs-deployment/multicluster/openshift_dr_cluster_operator.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+data:
+  ramen_manager_config.yaml: |
+    apiVersion: ramendr.openshift.io/v1alpha1
+    kind: RamenConfig
+    leaderElection:
+      resourceName: dr-cluster.ramendr.openshift.io
+    ramenControllerType: "dr-cluster"
+    s3StoreProfiles:
+      - s3ProfileName: odr-s3secret-east
+        s3CompatibleEndpoint: https://s3.us-east-2.amazonaws.com:443
+        s3Region: us-east-2
+        s3SecretRef:
+          name: odr-s3secret-east
+          namespace: openshift-dr-system
+      - s3ProfileName: odr-s3secret-west
+        s3CompatibleEndpoint: https://s3.us-east-2.amazonaws.com:443
+        s3Region: us-east-2
+        s3SecretRef:
+          name: odr-s3secret-west
+          namespace: openshift-dr-system
```

Comment:
There is a plan to use MCG only and not use AWS

Reply:
@prsurve I can handle this in next PR. Also, I need to get handson for how to configure MCG for this purpose. is that fine ?